src/activities/GenerateQueryWhereClause.ts

```diff
@@ -143,12 +157,22 @@ export default class GenerateWhereClause implements IActivityHandler {
         field: __esri.Field,
     ): string | undefined {
         let formattedValue;
+
         switch (field.type) {
             case EsriFieldType.Guid:
             case EsriFieldType.GlobalId:
             case EsriFieldType.String:
                 if (Array.isArray(value)) {
-                    formattedValue = value.map(x => `'${x}'`).join(",");
+
+                    formattedValue = value.map((x) => {
+                        if (x === null) {
+                            return "NULL";
```

Comment:
I think we should revert the code paths that return `"NULL"` for `null`. You will likely need to also change the return type from `string | undefined` to `string | null | undefined`. I see on line 175 you're already setting it to null. Maybe it's just accepting it because `formattedValue` is assumed to be of type `any`.